koku/masu/external/downloader/azure/azure_report_downloader.py

```diff
@@ -29,6 +29,7 @@
 from masu.external.downloader.downloader_interface import DownloaderInterface
 from masu.external.downloader.report_downloader_base import ReportDownloaderBase
 from masu.util.aws.common import copy_local_report_file_to_s3_bucket
+from masu.util.aws.common import get_path_prefix
```

Comment:
small nit.  Does it make sense to put these functions in `masu.util.common` since they are used in all downloaders?

Reply:
on second look.. I can see the reason for AWS now.  Copying to S3 is an AWS thing...

Reply:
Sure, could move them. I think that's a fair point.

I was thinking of it more regarding AWS S3 tech, than how its related to the provider types.

Reply:
yeah I can see that.  Maybe split the difference? Put `get_path_prefix` in common and leave `copy_local_report_file_to_s3_bucket` in the aws util?

Reply:
Done.